ui/src/app/pages/homepage/homepage.tsx

```diff
@@ -22,6 +22,7 @@ import colors, {addOpacity} from 'app/styles/colors';
 import {hasRegisteredAccessFetch, reactStyles, ReactWrapperBase, withUserProfile} from 'app/utils';
 import {AnalyticsTracker} from 'app/utils/analytics';
 import {fetchWithGlobalErrorHandler} from 'app/utils/retry';
+import {environment} from 'environments/environment';
```

Comment:
This change feels in the opposite direction from changes in all the other files (where we remove an `environment` dependency and add a `zendesk` dependency). See my comment on `environment.ts` below... it may be cleanest to make the env variable a simple prod vs. sandbox signal, and have all access to Zendesk URLs go through the zendesk module. Which would turn this reference into:

`href={supportUrls.helpCenterUrl}`

Reply:
Done per other suggestion. The reason this looked different is b/c the Zendesk URL was simply hardcoded on this page originally, i.e. in worse shape than the other links - otherwise I wouldn't have needed to change this